src/views/Play/Play.tsx

```diff
@@ -591,6 +591,9 @@ export class Play extends React.Component<PlayProperties, any> {
 
         let challengeList = isLive ? this.state.live_list : this.state.correspondence_list;
 
+        challengeList.forEach(element => {
+            console.log("komi: " + element.komi);
```

Comment:
Probably don't need this log anymore for production

Reply:
Oh yeah, I will (or you can) get rid of that :)